protocol_grpc.go

```diff
@@ -768,18 +723,29 @@ func grpcHTTPToCode(httpCode int) Code {
 // binary Protobuf format, even if the messages in the request/response stream
 // use a different codec. Consequently, this function needs a Protobuf codec to
 // unmarshal error information in the headers.
+//
+// A non-nil error is only returned when a grpc-status key IS present, but it
+// indicates a code of zero (no error). If no grpc-status key is present, this
+// returns a non-nil *Error that wraps errTrailersWithoutGRPCStatus.
```

Comment:
I'm having a hard time understanding this comment. Should it look like this instead?

```suggestion
// A nil error is only returned when a grpc-status key IS present, but it
// indicates a code of zero (no error). If no grpc-status key is present, this
// returns a non-nil *Error that wraps errTrailersWithoutGRPCStatus.
```

Reply:
Doh! You're absolutely right. My typing fingers seem to have reversed the intent of my brain 😂